logserver/pom.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+    (the "License"). You may not use this work except in compliance with the License, which is
+    available at www.apache.org/licenses/LICENSE-2.0
+
+    This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+    either express or implied, as more fully set forth in the License.
+
+    See the NOTICE file distributed with this work for information regarding copyright ownership.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>alluxio-parent</artifactId>
+        <groupId>org.alluxio</groupId>
+        <version>1.6.0-SNAPSHOT</version>
+    </parent>
+    <artifactId>alluxio-logserver</artifactId>
+    <packaging>jar</packaging>
+    <name>Alluxio Log Server</name>
+    <description>A logging server receiving log messages from Alluxio servers</description>
+
+    <properties>
+        <!-- The following paths need to be defined here as well as in the parent pom so that mvn can -->
+        <!-- run properly from sub-project directories -->
+        <license.header.path>${project.parent.basedir}/build/license/</license.header.path>
+        <checkstyle.path>${project.parent.basedir}/build/checkstyle/</checkstyle.path>
+        <findbugs.path>${project.parent.basedir}/build/findbugs/</findbugs.path>
+        <failIfNoTests>false</failIfNoTests>
+    </properties>
+
+    <dependencies>
+        <!-- External dependencies -->
+        <!-- Internal dependencies -->
+        <dependency>
+            <groupId>org.alluxio</groupId>
+            <artifactId>alluxio-core-server-common</artifactId>
+            <version>${project.version}</version>
+        </dependency>
```

Comment:
Please explicitly state all the dependencies ie. log4j, guava

Reply:
Done.